logstash-core/lib/logstash/environment.rb

```diff
@@ -88,6 +88,7 @@ module Environment
            Setting::Boolean.new("api.ssl.enabled", false),
   Setting::ExistingFilePath.new("api.ssl.keystore.path", nil, false).nullable,
           Setting::Password.new("api.ssl.keystore.password", nil, false).nullable,
+          Setting::SplittableStringArray.new("api.ssl.supported_protocols", nil, true, %w[TLSv1 TLSv1.1 TLSv1.2 TLSv1.3]),
```

Comment:
I don't think we need to support a comma-separated string on top of the expected array of strings, so we can switch to ArrayCoercible instead.

Reply:
Thanks for reviewing, sure! I'll change it to use the `ArrayCoercible`. 
Regarding the validation of the possible values implemented on this PR, do you think it would be useful to the `ArrayCoercible` class? if we want to keep the refactoring at the minimum I can pass a `validator_proc` instead.

Reply:
> Regarding the validation of the possible values implemented on this PR, do you think it would be useful to the ArrayCoercible class?

Either creating a StringArray setting class or passing a validator_proc should be fine.